hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -65,18 +65,26 @@
 public class HiveTableOperations extends BaseMetastoreTableOperations {
   private static final Logger LOG = LoggerFactory.getLogger(HiveTableOperations.class);
 
+  private static final String HIVE_ACQUIRE_LOCK_STATE_TIMEOUT_MS = "hive.lock.acquire-timeout-ms";
+  private static final long   HIVE_ACQUIRE_LOCK_STATE_TIMEOUT_MS_DEFAULT = 3 * 60 * 1000; // 3 minutes
+
   private final HiveClientPool metaClients;
   private final String database;
   private final String tableName;
   private final Configuration conf;
 
+  private final long lockAcquireTimeout;
+
   private FileIO fileIO;
 
   protected HiveTableOperations(Configuration conf, HiveClientPool metaClients, String database, String table) {
     this.conf = conf;
     this.metaClients = metaClients;
     this.database = database;
     this.tableName = table;
+
```

Comment:
Same here, I don't think we need to separate the lock timeout setting from the other fields that are getting initialized.